criu/arch/aarch64/include/asm/vdso.h

```diff
@@ -9,6 +9,7 @@
  * This is a minimal amount of symbols
  * we should support at the moment.
  */
+#define no_binary_log
```

Comment:
@cyrillos , @0x7f454c46 , is vdso.h the correct place for such a thing?

Reply:
I don't think so. Also, double-negative sounds hard to read:
`#ifndef no_binary_log`
And as it's a macro it should be in CAPITAL LETTERS.
If I would do something like that, I'd add:
```
ifeq ($(ARCH),x86)
        LDARCH		:= i386:x86-64
        DEFINES		:= -DCONFIG_X86_64 -DARCH_HAS_FLOG
endif
```
And so on to the supported arches. That way one can disable compilation of something that's not working anyway.

Furthermore, I don't understand what does `aarch64 and arm architectures are not compatible with the decoder`.
Is it something fixable? What means `not compatible`? What are the errors if you try enabling it? Is there a plan to support it?
Probably, with such definition should go a descriptive comment..

Reply:
Thank you, I will make these changes.

The line that causes issues with the decoder is this:
`size  = vsnprintf(buffer + buf_off, sizeof buffer - buf_off, fmt, (void *)values);`
 
The other architectures accept va_list in type (void *) but these arm and aarch64 architectures produce errors of an incompatible type. To fix this issue, there needs to be another way to feed the printf family function a variable number of variables in a way that is compatible with these architectures. 

I will also add this description to the commit message.

Reply:
Hey, I've looked at the result..
Let me try to help you a bit here from the makefile side:
https://github.com/0x7f454c46/criu/commit/6a659be02e56a06e84587a1937e594cb2eeee7ea
Please, reuse/integrate the changes in your commits: this way arm/aarch64 will skip compiling criu/flog.o and you won't need those scary undefs. I've also removed in that commit compiling crit/decoder for unsupported case, but if you feel like keeping it for consistency - I don't insist, just return ifdef CONFIG_HAS_FLOG there.
Thanks!

Reply:
Thanks a lot! I will integrate these changes and send the update by tonight.
I believe the crit/decoder would be needed for the other unsupported cases as well because zdtm.py generates calls for them. Would you suggest changing zdtm.py for unsupported cases, or is it better to retain the decoder?